cmd/ocm-backplane/console/console_test.go

```diff
@@ -141,31 +117,42 @@ var _ = Describe("console command", func() {
 			setupConfig()
 
 			mockOcmInterface.EXPECT().GetOCMAccessToken().Return(&testToken, nil).AnyTimes()
+			mockOcmInterface.EXPECT().GetPullSecret().Return(&pullSecret, nil).AnyTimes()
+			mockOcmInterface.EXPECT().GetClusterInfoByID(clusterID).Return(&clusterInfo, nil).AnyTimes()
```

Comment:
I changed it to clusterinfor and got the below error
```
• Failure [0.001 seconds]
console command
/Users/revyas/SRE/github/backplane-cli/cmd/ocm-backplane/console/console_test.go:24
  when kube config is invalid
  /Users/revyas/SRE/github/backplane-cli/cmd/ocm-backplane/console/console_test.go:165
    should start not console server [It]
    /Users/revyas/SRE/github/backplane-cli/cmd/ocm-backplane/console/console_test.go:166

    Unexpected call to *mocks.MockOCMInterface.GetClusterInfoByID([configcluster]) at /Users/revyas/SRE/github/backplane-cli/cmd/ocm-backplane/console/console.go:280 because:
    expected call at /Users/revyas/SRE/github/backplane-cli/cmd/ocm-backplane/console/console_test.go:172 doesn't match the argument at index 0.
    Got: configcluster (string)
    Want: is equal to  (string)
```